app/src/main/java/com/example/development/sakaiclientandroid/api_models/gradebook/GradebookCollectionObject.java

```diff
@@ -37,37 +32,7 @@ public String getSiteId() {
         return siteId;
     }
 
-    public void setSiteId(String siteId) {
-        this.siteId = siteId;
-    }
+    public String getSiteName() { return siteName; }
```

Comment:
I was curious, does getting rid of the setter still allow the getter to work as intended? Can you check if this still holds true, because I would think GSON might need _some_ public-facing method to be able to set the value in the first place.

Reply:
Yes I removed the setters and it still worked, I think GSON is using the @Expose annotation to put the value in that field

Reply:
Oh sweet, that's really nice actually because the encapsulation is better and we can't accidentally change a parsed value. This was my main concern, so I'll merge the PR now.